DigitalLearningSolutions.Web/Helpers/DisplayStringHelper.cs

```diff
@@ -17,7 +17,7 @@ public static string FormatBytesWithLimit(long number, long limit)
             return GenerateBytesDisplayString(number) + Divider + GenerateBytesDisplayString(limit);
         }
 
-        private static string GenerateBytesDisplayString(long byteCount)
+        public static string GenerateBytesDisplayString(long byteCount)

```

Comment:
What kind of output does this give us if I put in something like `477210`?

Reply:
Just checking the name makes sense in an external context.

Reply:
466KB

Reply:
I wonder if a more understandable interface would be `GenerateSizeDisplayString(long sizeInBytes)`. "Bytes" can be displayed / interpreted in a few different ways - signed or unsigned, or as characters after being sent through a character encoding. This name is worth getting right because the logic in the method is very mathsy and hard to get your head around. What do you think?